src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,95 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.InvalidUserDataException;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationServiceImpl registrationService;
+
+    @BeforeAll
+    static void setUp() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @Test
+    public void register_NullUser_ThrowsException() {
+        assertThrows(InvalidUserDataException.class, () -> {
+            registrationService.register(null);
+        });
+    }

```

Comment:
fix the naming all methods
```suggestion
    public void register_nullUser_notOk() {
        assertThrows(InvalidUserDataException.class, () -> {
            registrationService.register(null);
        });
    }

```